sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -1659,6 +1659,21 @@ case class LocalLimit(limitExpr: Expression, child: LogicalPlan) extends OrderPr
     copy(child = newChild)
 }
 
+/** Logical node that represents the LIMIT ALL operation. This operation is usually no-op and exists
+ *  to provide compatability with other databases. However, in case of recursive CTEs, Limit nodes
```

Comment:
```suggestion
/** 
 * Logical node that represents the LIMIT ALL operation. This operation is usually no-op and exists
 *  to provide compatability with other databases. However, in case of recursive CTEs, Limit nodes
```